content-src/components/TopSites/TopSites.js

```diff
@@ -25,20 +25,23 @@ const TopSitesItem = React.createClass({
     const site = this.props;
     const index = site.index;
     const isActive = this.state.showContextMenu && this.state.activeTile === index;
-    return (<div className={classNames("tile-outer", {active: isActive})} key={site.guid || site.cache_key || index}>
-      <a onClick={() => this.props.onClick(index)} className="tile" href={site.url} ref="topSiteLink">
-        <SiteIcon className="tile-img-container" site={site} faviconSize={32} showTitle={true} />
-        <div className="inner-border" />
-      </a>
-      <LinkMenuButton onClick={() => this.setState({showContextMenu: true, activeTile: index})} />
-      <LinkMenu
-        visible={isActive}
-        onUpdate={val => this.setState({showContextMenu: val})}
-        site={site}
-        page={this.props.page}
-        source="TOP_SITES"
-        index={index} />
-  </div>);
+
+    let className = classNames("tile-outer", {active: isActive});
+    return (
+      <div className={className} key={site.guid || site.cache_key || index}>
+        <a onClick={() => this.props.onClick(index)} className="tile" href={site.url} ref="topSiteLink">
+          <SiteIcon ref="icon" className="tile-img-container" site={site} faviconSize={32} showTitle={true} />
+          <div className="inner-border" />
+        </a>
+        <LinkMenuButton onClick={() => this.setState({showContextMenu: true, activeTile: index})} />
+        <LinkMenu
+          visible={isActive}
+          onUpdate={val => this.setState({showContextMenu: val})}
+          site={site}
+          page={this.props.page}
+          source="TOP_SITES"
+          index={index} />
+      </div>);
```

Comment:
I've undone everything except the fact that I had to add a ref="icon" for testing.